src/System.DirectoryServices/ref/Configurations.props

```diff
@@ -2,8 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netcoreapp;
-      uap;
+      ;
```

Comment:
yes. We haven't really added anything to these `DirectoryServices` assemblies yet, so it makes no sense to have them building on any framework. Plus, the code on the implementation side that we do have doesn't compile yet, so build is disabled there as well.